test/conftest.py

```diff
@@ -248,8 +250,12 @@ def pudl_engine(ferc1_engine, live_pudl_db, pudl_settings_fixture,
 
 
 @pytest.fixture(scope='session')  # noqa: C901
-def pudl_settings_fixture(request, tmpdir_factory,  # noqa: C901
-                          live_ferc1_db, live_pudl_db):  # noqa: C901
+def pudl_settings_fixture(  # noqa: C901
+    request,  # noqa: C901
+    tmpdir_factory,  # noqa: C901
+    live_ferc1_db,  # noqa: C901
+    live_pudl_db,  # noqa: C901
+):  # noqa: C901
```

Comment:
do these really need to be on every line? that seems excessive.

also we really should break up this function although that is clearly out of scope of this PR.

Reply:
Not sure. I was just copying the formatting of the function as I found it.